### ISSUE TITLE
Fix table header for "contributors" table

### ISSUE THREAD

**Issue by grinnellm** [grinnellm/csas-style-sr#4]
The table of contributors fails the "Web Accessibility" check for tables, which must have a defined table header (i.e., column names).

**Comment by ricardd** [comment]
Hey Matt, who runs this "Web Accessibility" check on the resulting PDF? Is this something that we can do ourselves, or does CSAS do that?

**Comment by grinnellm** [comment]
If you have Adobe Pro you can check it yourself --follow the instructions for [csas-latex here](https://github.com/pbs-assess/csas-style).
